drivers/espi/espi_emul.c

```diff
@@ -212,41 +212,31 @@ int espi_emul_register(const struct device *dev, const char *name,
 
 /* Device instantiation */
 static struct emul_espi_driver_api emul_espi_driver_api = {
-	.espi_api = {
-		.config = espi_emul_config,
-		.get_channel_status = espi_emul_get_channel_status,
-		.read_lpc_request = espi_emul_read_lpc_request,
-		.write_lpc_request = espi_emul_write_lpc_request,
-		.send_vwire = espi_emul_send_vwire,
-		.receive_vwire = espi_emul_receive_vwire,
-		.manage_callback = espi_emul_manage_callback
-	},
+	.espi_api = { .config = espi_emul_config,
```

Comment:
nit: formatting here looks odd.  the `.config...` seems like it should be on its own line.

Reply:
This is from [a clang-format](https://github.com/zephyrproject-rtos/zephyr/blob/main/.clang-format), I tried putting the `.config...` on its own line and `clang-format` just puts it back where it is now.